src/main/java/com/es/phoneshop/service/productHistory/CustomProductHistoryService.java

```diff
@@ -24,19 +24,15 @@ public static CustomProductHistoryService getInstance() {
     @Override
     public ProductHistory getProductHistory(HttpServletRequest request) {
```

Comment:
let's improve namings a bit
productHistory looks confusing, maybe smth like getViewedProductsHistory, to epmhasize that it's a list of visited products